specs/CalendarView/CalendarViewSpec1.md

```diff
@@ -0,0 +1,498 @@
+
+### Showcasing the BlackoutStrikethroughBrush property - all Sundays are blacked out
+![Blacked Out Sundays.](images/BlackoutStrikethroughBrush.png)
+
+## Out Of Scope styling
+The following properties of type Windows.UI.Xaml.Media.Brush are used to customize the calendar items that are outside the current scope (month, year, or decade):
+OutOfScopeHoverForeground, OutOfScopePressedForeground.
+
+### Showcasing the OutOfScopeHoverForeground property - 4th of May is hovered
+![Hover Out Of Scope Foreground.](images/HoverOutOfScopeForeground.png)
+
+### Showcasing the OutOfScopePressedForeground property - 4th of May is pressed
+![Pressed Out Of Scope Foreground.](images/PressedOutOfScopeForeground.png)
+
+## Item background styling
+The following properties of type Windows.UI.Xaml.Media.Brush are used to customize the background rendering of items in any DisplayMode:
+CalendarItemHoverBackground, CalendarItemPressedBackground.
+
+### Showcasing the CalendarItemHoverBackground property - February is hovered
+![Hover Item Background.](images/HoverItemBackground.png)
+
+### Showcasing the CalendarItemPressedBackground property - 2034 is pressed
+![Pressed Item Background.](images/PressedItemBackground.png)
+
+## Disabled styling
+The following properties of type Windows.UI.Xaml.Media.Brush are used to customize the rendering of a disabled CalendarView (when CalendarView.IsEnabled is False):
+DisabledForeground, SelectedDisabledBorderBrush, CalendarItemDisabledBackground, TodayDisabledBackground, TodayDisabledForeground.
+
+### Showcasing a disabled month view - the 12th and 20th of April are selected
+![A disabled CalendarView.](images/DisabledCalendarView.png)
+
+## Day item positioning
+The CalendarView's DayItemMargin, MonthYearMargin, FirstOfMonthLabelMargin, FirstOfYearDecadeLabelMargin properties of type Windows.UI.Xaml.Thickness 
+allow positioning the day, month, year, first-of-month and first-of-year labels within a calendar item. 
+
+### Showcasing the DayItemMargin and FirstOfMonthLabelMargin properties
+Here the Month view has CalendarView.IsGroupLabelVisible set to True and the group and main labels do not overlap because CalendarView.DayItemMargin.Top is set to 6 pixels.
+April 1st is selected and May 1st is hovered.
+
+![A month view with visible group labels.](images/MonthViewWithVisibleGroupLabels.png)
+
+### Showcasing the MonthYearMargin and FirstOfYearDecadeLabelMargin properties
+A Year view has CalendarView.IsGroupLabelVisible set to True with CalendarView.MonthYearItemMargin.Top and CalendarView.FirstOfYearDecadeLabelMargin.Top set to 2 and 3 pixels respectively.
+January 2021 is hovered.
+
+![A year view with visible group labels.](images/YearViewWithVisibleGroupLabels.png)
+
+## Markup example
+
+This shows a typical use of the new CalendarView properties set in markup. The HighContrast & Dark themes and control template have been removed for conciseness.
```

Comment:
Is this typical or default? (One thing it is is long.) Would an app really typically do the resource indirection?

Reply:
This is a forecast of what our WinUI 2.6 default styling will look like. An app would typically just use <CalendarView .../>.